test/const.go

```diff
@@ -0,0 +1,27 @@
+package test
+
+import "time"
+
+const (
+	// DefaultSuccessTimeout for operations that are expected to succeed.
+	DefaultSuccessTimeout = 5 * time.Minute
+
+	// DefaultFailureTimeout for operations that are *expected* to time out.
+	//
+	// The timeout is short because timeouts are *never* a reliable way to
+	// determine that something does not happen. At best such a test *might* catch
+	// the problem. It is better than nothing, but not by much. Definitely not
+	// worth delaying the entire test suite for.
+	//
+	// Ideally tests for non-happening should have a positive wait with
+	// DefaultSuccessTimeout for something that we know *does* happen, and that we
+	// know never happens *before* the thing we don't expect to happen.  Then look
+	// for positive proof that the thing didn't happen. A negative timeout result
+	// never proves that the thing won't happen in 1 more millisecond.
+	//
+	DefaultFailureTimeout = 1 * time.Second
+
+	InstanceName               = "instance"
+	OpenshiftLoggingNS         = "openshift-logging"
```

Comment:
:) circular dependency is exactly what happend. Since package test doesn't have any dependencies, I can re-define helpers/constants.go in terms of the test/const.go so there's only one definition.